tests/features/authentication.feature

```diff
@@ -44,13 +41,10 @@ Feature: User authentication
       | rdf_entity/add/solution        |
 
   @api
-  Scenario Outline: Authenticated user can access pages they are authorized to
-    Given I am logged in as a user with the "authenticated" role
-    Then I visit "<path>"
-
-    Examples:
-      | path                           |
-      | rdf_entity/http%3A\\drupal.org |
+  Scenario: Authenticated user can access pages they are authorized to
+    Given I am logged in as a user with the "authenticated user" role
+    When I go to the homepage of the "Überwaldean Land Eels" collection
+    Then I should see the heading "Überwaldean Land Eels"
```

Comment:
Revert this, this doesn't belong here, this is the generic authentication test for administration pages. If you want to test collections, do this in one of the tests in the `tests/features/collection/` folder.